packages/app/src/components/DeleteCredentialSheet.tsx

```diff
@@ -0,0 +1,84 @@
+import { usePidCredential } from '@easypid/hooks'
+import { type CredentialForDisplayId, deleteCredential, useAgent } from '@package/agent/src'
+import { Heading, HeroIcons, Paragraph, Stack, XStack, useToastController } from '@package/ui'
+import { FloatingSheet } from '@package/ui/src/panels/FloatingSheet'
+import { useState } from 'react'
+import { useHaptics } from '../hooks'
+import { DualResponseButtons } from './DualResponseButtons'
+
+interface DeleteCredentialSheetProps {
+  isSheetOpen: boolean
+  setIsSheetOpen: (isOpen: boolean) => void
+  id: CredentialForDisplayId
+  name: string
+}
+
+export function DeleteCredentialSheet({ isSheetOpen, setIsSheetOpen, id, name }: DeleteCredentialSheetProps) {
+  const { credential } = usePidCredential()
+  const toast = useToastController()
+  const { agent } = useAgent()
+  const [isLoading, setIsLoading] = useState(false)
+  const { withHaptics, error, success } = useHaptics()
+
+  const onDeleteCredential = async () => {
+    if (credential && id === credential?.id) {
```

Comment:
Do we use the same detail screen for both? I thought we had a different detila screen for the pid? In that case we can just not render the delete sheet right? I think this is OK, but it's a bit hacky as a very low-level component now does PID specific logic, while it would make more sense to just not render / or pass props from higher up that this credential can't be archived